market/connector/currencyConverter/meta.json

```diff
@@ -0,0 +1,58 @@
+{
+	"id": "currencyConverter-connector",
+	"name": "Currency Converter Connector",
```

Comment:
just "Currency Converter" should be enough as name. I think by the tag 'connector' it's already marked as such and this additional postfix just makes the name longer and harder to read on smaller browsers

Reply:
Yes, please remove connector from ID and name.

Reply:
Maybe it should also have its official name: "Frankfurter Currency Data API"